### DISCUSSION TITLE
Change default of which "special key" gets sent from "Mapper 'send special key'" menu item?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4433] [Q&A] [answered]
From the DOSBox-X Main menu, `Send special key` starts with `Mapper "send special key": Ctrl-Alt-Del` checked by default. I can't find anything in the configuration that lets me specify that a different key can be checked by default. Am I missing something obvious?

**Comment by maron2000** [comment, marked as answer]
You can find `mapper send key` in the conf file.

**Comment by emendelson** [comment]
I should have found that by myself. Thank you!
